packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Snippets/ArgumentSnippetsTests.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Generator.CSharp.Tests.Snippets
+{
+    internal class ArgumentSnippetsTests
```

Comment:
was adding this empty class intentional ?

Reply:
no, this was copy paste error, good catch